### PR TITLE
INTYGFV-16109: Display preamble text from getCertificate response.

### PR DESCRIPTION
**Pull request by ChrilleSigerhed**

### CODE REVIEW

**Comment by ChrilleSigerhed** [comment]
image

File: apps/minaintyg/src/schema/certificate.schema.ts
Comment (SamuelTilly) [inline apps/minaintyg/src/schema/certificate.schema.ts:1]: Please remove `description` from `certificateTypeSchema`

File: apps/minaintyg/src/mocks/handlers.ts [hunk 1]
```diff
@@ -121,6 +122,7 @@ export const handlers = [
             information: [],
           }),
         ],
+        texts: fakerFromSchema(certificateTextSchema)({ PREAMBLE_TEXT: 'Ingresstext' }),
```
Comment (SamuelTilly) [inline apps/minaintyg/src/mocks/handlers.ts:125]: Could use the mock preamble text with `{PREAMBLE_TEXT: certificateIngress(certificate.id) ?? 'Ingresstext'}`

File: apps/minaintyg/src/schema/certificate.schema.ts [hunk 2]
```diff
@@ -63,11 +63,15 @@ export const availableFunctionSchema = z.object({
   information: z.array(informationSchema),
 })
 
+export const certificateTextSchema = z.object({
+  PREAMBLE_TEXT: z.string(),
```
[the remark this answers is not in the document]
Comment (ChrilleSigerhed) [reply apps/minaintyg/src/schema/certificate.schema.ts]: Updated to z.record

**Review by SamuelTilly — APPROVED** [review]

**Pull request merged**